### PR TITLE
[16.0][MIG] website_sale_wishlist_keep: Migration to version 16.0

### PR DESCRIPTION
**Pull request by pilarvargas-tecnativa**
cc @Tecnativa TT44389

@CarlosRoca13 @chienandalu please review :)

### CODE REVIEW

File: website_sale_wishlist_keep/static/src/js/tour.js [hunk 1]
```diff
@@ -4,12 +4,13 @@
 odoo.define("website_sale_wishlist_keep.tour", function (require) {
     "use strict";
 
-    const tour = require("web_tour.tour");
-    const base = require("web_editor.base");
+    var tour = require("web_tour.tour");
 
-    const steps = [
+    var steps = [
```
Comment (chienandalu) [inline website_sale_wishlist_keep/static/src/js/tour.js]: why?

**Review by chienandalu — CHANGES_REQUESTED** [review]
:+1: Tested ok

Minor request :point_down:

File: website_sale_wishlist_keep/static/src/js/tour.js (hunk 1 above)
Comment (pilarvargas-tecnativa) [reply website_sale_wishlist_keep/static/src/js/tour.js]: A test I had forgotten to reinstate but it wasn't the access problem that one, thanks!

**Review by CarlosRoca13 — APPROVED** [review]

**Review by chienandalu — APPROVED** [review]

**Pull request merged**